exporter/lokiexporter/exporter.go

```diff
@@ -35,14 +36,23 @@ type lokiExporter struct {
 	settings component.TelemetrySettings
 	client   *http.Client
 	wg       sync.WaitGroup
+
+	lokiExporterFailedToSendLogRecordsDueToMissingLabels metric.Int64Counter
 }
 
 func newExporter(config *Config, settings component.TelemetrySettings) *lokiExporter {
 	settings.Logger.Info("using the new Loki exporter")
 
+	count, _ := metadata.Meter(settings).Int64Counter(
```

Comment:
I would not ignore the error if possible (a.k.a not too many changes to return error in newExporter).

Reply:
added error handling